### ISSUE TITLE
Add compatibility with codesandbox.io

### ISSUE THREAD

**Issue by jmorel** [davidhu2000/react-spinners#56]
**Do you want to request a feature or report a bug?**
Feature

**If reporting a bug, what version of `react-spinners` are you currently using?**
0.5.1

**What is the current behavior?**

Adding react-spinners 0.5.1 as a dependency of a react project on codesandbox.io raises an error during the build of the project. This prevents react-spinners from being part of any example/project hosted there.

The error:

```
Error
Could not fetch dependencies, please try again in a couple seconds: Something went wrong while packaging the dependency react-spinners@0.5.1: ENOENT: no such file or directory, scandir '/tmp/3665931091/node_modules/react-spinners/dist'
```

See example: https://codesandbox.io/s/kko6481k3r

**Comment by davidhu2000** [comment]
The reason it's not currently working is because the outputted js lives in the root folder, not `dist`. Is there a way to change that on codesandbox? So it imports from the root instead of `dist`? 

The reason behind deploying the files in the root is to allow individual imports to help reduce import cost, like 

`import BarLoader from 'react-spinners/BarLoader';`

**Comment by jmorel** [comment]
Following the fix in #81 it now works like a charm, thanks!
https://codesandbox.io/s/affectionate-mendel-rm2ku

**Comment by davidhu2000** [comment]
Awesome. i'll create a codesandbox account and setup a demo there.
